client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -125,6 +167,13 @@ private void initializeLifecycleManager() {
       synchronized (this) {
         if (lifecycleManager == null) {
           lifecycleManager = new LifecycleManager(appUniqueId, celebornConf);
+          if (celebornConf.clientFetchThrowsFetchFailure()) {
+            MapOutputTrackerMaster mapOutputTracker =
+                (MapOutputTrackerMaster) SparkEnv.get().mapOutputTracker();
```

Comment:
Why convert to `MapOutputTrackerMaster`?

Reply:
this function is executed in spark driver that has a MapOutputTrackerMaster, and the method unregisterAllMapAndMergeOutput is a member of MapOutputTrackerMaster not MapOutputTracker